### PR TITLE
Add statuses path to service model

### PR DESCRIPTION
**Pull request by djberg96**
This adds a missing statuses endpoint to the services path, plus a few comments.

### CODE REVIEW

File: model/status_board/v1/product_resource.model [hunk 1]
```diff
@@ -27,10 +27,12 @@ resource Product {
   method Delete {
   }
 
+  // /api/status-board/v1/products/{id}/applications
```
Comment (jhernand) [inline model/status_board/v1/product_resource.model:30]: I think it is redundant to have this here, as this is automatically calculated by the model and added to the generated OpenAPI documentation. If you really want to include it then add a first sentence explaining that this is, and make this a complete example:

```
// Reference to the service that manages the collection
// of applications associated to this product. For example
// go get the list of applications for product `123` send a
// request like this:
//
// ```http
// GET /api/status-board/v1/products/123/applications
// ```
```

File: model/status_board/v1/product_resource.model [hunk 2]
```diff
@@ -27,10 +27,12 @@ resource Product {
...
   locator Applications {
     target Applications
   }
 
+  // /api/status-board/v1/products/{id}/updates
```
Comment (jhernand) [inline model/status_board/v1/product_resource.model:35]: Same.

File: model/status_board/v1/service_resource.model [hunk 3]
```diff
@@ -26,4 +26,9 @@ resource Service  {
 
   method Delete {
   }
+
+  // /api/status-board/v1/services/{id}/statuses
```
Comment (jhernand) [inline model/status_board/v1/service_resource.model:30]: Same.

**Comment by djberg96** [comment]
@jhernand Ok, updated.

**Review by jhernand — APPROVED** [review]

**Pull request merged**